Authenticator/Source/OTPTokenListViewController.swift

```diff
@@ -261,6 +268,8 @@ extension OTPTokenListViewController /* UITableViewDelegate */ {
             let section = self.numberOfSectionsInTableView(self.tableView) - 1
             let row = self.tableView(self.tableView, numberOfRowsInSection: section) - 1
             self.tableView.scrollToRowAtIndexPath(NSIndexPath(forRow: row, inSection: section), atScrollPosition: .Middle, animated: true)
+        } catch {
```

Comment:
Length Violation: Line should be 100 characters or less: currently 138 characters